packages/react-email/package.json

```diff
@@ -28,26 +39,41 @@
     "@commander-js/extra-typings": "9.4.1",
     "@manypkg/find-root": "2.2.1",
     "@octokit/rest": "19.0.7",
+    "@react-email/components": "workspace:*",
     "@react-email/render": "0.0.11-canary.0",
+    "@types/node": "20.10.4",
+    "@types/react": "18.2.33",
+    "@types/react-dom": "18.2.14",
+    "autoprefixer": "10.4.14",
+    "chalk": "5.3.0",
     "chokidar": "3.5.3",
     "commander": "9.4.1",
     "detect-package-manager": "2.0.1",
     "esbuild": "0.16.4",
     "fs-extra": "11.1.1",
     "glob": "10.3.4",
     "log-symbols": "4.1.0",
+    "next": "14.0.4",
     "normalize-path": "3.0.0",
     "ora": "5.4.1",
+    "postcss": "8.4.32",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
I wonder if we should somehow allow a specific range instead of a fixed version, not sure if eventually React version will be a compatibility problem when people install react-email in their repos

Reply:
Maybe, Next uses `^18.2.0` instead of an exact version, maybe we should do the same 🤔